alchemiscale/tests/integration/storage/test_statestore.py

```diff
@@ -1019,6 +1093,15 @@ def test_get_task_transformation(
         assert tf == transformation
         assert tf_sk == transformation_sk
 
+    def test_get_scope_status(self, n4js: Neo4jStore):
```

Comment:
Same here, this PR or another one?

Reply:
I'll fill these in now. I prioritized getting the `AlchemiscaleClient` tests in for the API points, since these end up exercising the `Neo4jStore` methods anyway, though that will be invisible to codecov due to how they get called.